examples/features/dual-stack/Kernel2Wireguard2Kernel_dual_stack/README.md

```diff
@@ -9,8 +9,7 @@ Forwarders are using the `wireguard` mechanism to connect with each other.
 
 Create test namespace:
 ```bash
-NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/3d1dcfe1de90681213c7f0006f25279bb4699966/examples/features/namespace.yaml)[0])
-NAMESPACE=${NAMESPACE:10}
+kubectl create ns ns-kernel2wireguard2kernel-dual-stack
```

Comment:
@NikitaSkrynnik wouldn't it be easier and more readable instead change just a value of NAMESPACE to ns-kernel2wireguard2kernel-dual-stack and use NAMESPACE later in the commands?

Reply:
We decided to get rid of NAMESPACE env completely